readinglist/resource.py

```diff
@@ -287,16 +398,28 @@ def collection_get(self):
 
         filters = self._extract_filters(self.request.GET)
         sorting = self._extract_sorting(self.request.GET)
-        records = self.db.get_all(filters=filters,
-                                  sorting=sorting,
-                                  **self.db_kwargs)
+        pagination_rules, limit = self._extract_pagination_token(
```

Comment:
Well the we extract the pagination_token into pagination rules why is that wrong?